interface/reports/patient_list_creation.php

```diff
@@ -96,7 +124,7 @@ function Form_Validate() {
                         return false;
                     }
                 }
-                $("#processing").show();
+                // $("#processing").show(); - Remains after CSV file is downloaded, removed temporarily
```

Comment:
Is this a bug that plan to fix in future?

Reply:
This is in essence either a bug or a feature removal. Other report types with the ability to export to CSV do not have the processing wheel, this is an issue unique to Patient List Creation. We can find a way to keep the wheel by preventing it from appearing when submitting a form via CSV export in future, but it is not a necessity.